packages/mapviewer/tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -506,6 +523,63 @@ describe('Testing the feature selection', () => {
             cy.get('@emptyIdentify.all').should('have.length', 1)
         })
 
+        it('can select feature by click, add more feature, and deselect feature', () => {
+            // Import KML file
+            const fileName = '4-points.kml'
+            const localKmlFile = `import-tool/${fileName}`
+            cy.goToMapView()
+            cy.wait(['@routeChange', '@layerConfig', '@topics', '@topic-ech'])
+
+            cy.openMenuIfMobile()
+            cy.get('[data-cy="menu-tray-tool-section"]:visible').click()
+            cy.get('[data-cy="menu-advanced-tools-import-file"]:visible').click()
+            cy.get('[data-cy="import-file-local-btn"]:visible').click()
+
+            cy.fixture(localKmlFile).as('kmlFile')
+            cy.get('[data-cy="file-input"]').selectFile(
+                { contents: '@kmlFile', fileName: fileName },
+                { force: true }
+            )
+
+            cy.get('[data-cy="import-file-load-button"]:visible').click()
+
+            cy.wait(['@icon-sets', '@icon-set-babs', '@icon-set-default'])
+
+            cy.get('[data-cy="file-input-text"]').should('contain.value', fileName)
+            cy.get('[data-cy="import-file-close-button"]:visible').click()
+            cy.readStoreValue('state.layers.activeLayers.length').should('eq', 1)
+            cy.readStoreValue('getters.visibleLayers.length').should('eq', 1)
+
+            cy.closeMenuIfMobile()
+
+            cy.checkOlLayer([
+                'test.background.layer2',
+                fileName,
+            ])
+
+            cy.get('[data-cy="ol-map"]').as('olMap').should('be.visible')
+
+            // This point location is found by listening to the click event on the map with the following code:
+            // cy.get('[data-cy="ol-map"]').then(($el) => {
+            //     $el[0].addEventListener('click', (e) => {
+            //         console.log('Clicked at:', e.offsetX, e.offsetY);
+            //     });
+            // });
+            const point1 = { x: 40, y: 205 }
+            const point3 = { x: 86, y: 359 }
```

Comment:
do you know your map coordinates for those points?

If so, you could use the OL instance to give you the correct pixel with [getPixelFromCoordinate](https://openlayers.org/en/latest/apidoc/module-ol_Map-Map.html#getPixelFromCoordinate)
that would be more resilient to change, especially when we will want to test bigger viewports